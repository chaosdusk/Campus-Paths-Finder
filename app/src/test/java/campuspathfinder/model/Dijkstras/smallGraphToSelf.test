# load small graph
LoadGraph small staffSuperheroes.tsv

# test to self
FindPath small Ernst-the-Bicycling-Wizard Ernst-the-Bicycling-Wizard