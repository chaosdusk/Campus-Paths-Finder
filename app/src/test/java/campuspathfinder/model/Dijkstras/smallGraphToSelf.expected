# load small graph
loaded graph small

# test to self
path from Ernst-the-Bicycling-Wizard to Ernst-the-Bicycling-Wizard:
total cost: 0.000